openiddict/src/Audacia.Auth.OpenIddict.Seeding.EntityFramework/EntityFrameworkSeeder.cs

```diff
@@ -29,7 +29,10 @@ public EntityFrameworkSeeder(string identityProjectBasePath, string identityProj
     protected override IServiceCollection ConfigureAdditionalServices(IServiceCollection services, IConfiguration configuration)
     {
         var connectionString = configuration.GetConnectionString(_connectionStringName);
-        services.AddScoped(_ => new OpenIddictContext<TKey>(connectionString));
+        if (connectionString != null)
```

Comment:
If we are checking if the connection string is not null, should we continue if it is null?